app/org/sagebionetworks/bridge/json/DateUtils.java

```diff
@@ -158,4 +158,25 @@ public static Long convertToMillisFromDuration(String duration) {
         return null;
     }
 
+    /**
+     * Convert at 8601 time zone offset string (e.g. "-07:00") to a DateTimeZone 
+     * object. 
+     * @param offsetString
+     * @return
+     */
+    public static DateTimeZone parseZoneFromOffsetString(String offsetString) {
```

Comment:
Strictly speaking, in ISO 8601, timezone offset strings, the colon is optional, the minutes are optional, and +00:00 is freqently represented as "Z". Do we want to handle those cases as well?

Maybe instead of writing our own string parsing, maybe it would be easier to just take a struct with the hour offset and minute offset as ints?